templates/repo_report_template.md

```diff
@@ -108,5 +108,10 @@ date_stampLastWeek: {date_stamp}
     <figure>
       <embed type="image/svg+xml" src="{{{{ "/assets/img/graphs/{repo_owner}/{repo_name}/commit_sparklines_{repo_name}_data.svg" | url }}}}" />
     </figure>
+    <div class="firstResponsePRCrop">
```

Comment:
Could you add this comment above the firstResponsePRCrop div:
`<!--- First Response For Closed PR Scatterplot -->`